crates/trie-parallel/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "reth-trie-parallel"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+description = "Parallel implementation of merkle root algorithm"
+
+[lints]
+workspace = true
+
+[dependencies]
+# reth
+reth-primitives.workspace = true
+reth-db.workspace = true
+reth-trie.workspace = true
+reth-provider.workspace = true
+reth-tasks.workspace = true
+
+# alloy
+alloy-rlp.workspace = true
+
+# tracing
+tracing.workspace = true
+
+# misc
+tokio = { workspace = true, default-features = false, features = ["sync"] }
```

Comment:
nit: introduce `async` and `parallel` features, gate tokio/rayon depending on the choice

Reply:
good point. i wanted to do this initially, but got lazy 😅